docs/pages/setup/security.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Security
+description: How to prevent attacks on your Teleport cluster
```

Comment:
I'd use something more generic: Hardening security of your Teleport deployment

Reply:
👍